docs/dispatch/Branches_and_Builds.md

```diff
@@ -70,89 +70,164 @@ dispatch branch list <application_id>
+`manifests` is an array of objects that denote file bundles; part of that object is listed here. `label` is the name you want to give an individual manifest/bundle of files. `platforms` are the platforms for which it is valid. `locales` is an array of locales for which the manifest is valid; leaving it empty denotes it's valid for all locales. `local_root` is the relative path from this file to the directory that contains the raw game files to upload. `redistributables` is an array of any redistributable packages your game may need to function, like a certain install of DirectX, or a Microsoft C++ redistributable. A list of valid values can be found in [Field Values](#DOCS_DISPATCH_FIELD_VALUES/).
 
-        // The locales the bundle supports
-        "locales": ["en-US", "en-GB"],
+```js
+{
+  "manifests": [
+    {
+      "file_rules": {
+        "mappings": [
+          {
+            "local_path": ".",
+            "install_path": "."
+          },
+          {
+            "local_path": "./languages/en-US/no-but-the-data-is-really-in-here/"
+            "install_path": "./english"
+          }
+        ],
+        "properties": [
+          {
+            "install_path": "save/*",
+            "attributes": ["user_data"]
+          }
+        ],
+        "exclusions": [
+          {
+            "local_path": "**/*.pdb"
+          },
+          {
+            "local_path": "server_distribution/linux/*"
+          },
+          {
+            "local_path": "client_distribution/linux/*"
+          },
+          {
+            "local_path": "client_distribution/osx/*"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
 
-        // the directory that contains the files for this bundle
-        "dir": "data/",
+This is another subset of a manifest object, `file_rules`. `file_rules` allows you to mark certain globs of files with certain tags, so Dispatch can handle them specially.
 
-        // Allows you to specify files that Dispatch should not touch on update or upload
-        "file_rules": {
+`mappings` lets you tell Dispatch to download files to a certain place in the install directory on a user's machine, letting you create the folder structure you need.
+
+`properties` allows you to mark properties on globs of files. In this case, marking a glob of files as `"user_data"` tells Dispatch not to touch these files in any way if it sees them; don't want that save data overwritten!
 
-          // Allows you to set file properties on directories for the installation
-          // Marking a folder as "user_data" means that Dispatch will never overwrite or otherwise touch these files
-          "properties" [
+`exclusions` also allow you to mark off globs of files. Files globs here will not be uploaded by Dispatch on a build push. In the above example, debug files that match the `*.pdb` pattern in any directory will be ignored. We also explicitly ignore linux and osx server and client distributables, since this manifest is for Windows.
```

Comment:
The manifest doesn't have Windows `platforms` even though the docs say it's a windows only manifest.

Reply:
Maybe because this is just a sub-section.

Reply:
Correct